helm/wfl/templates/configmap.yaml

```diff
@@ -6,6 +6,165 @@ metadata:
   labels:
     {{- include "wfl.labels" . | nindent 4 }}
 data:
+  override: |-
```

Comment:
Is there no other way of doing this? Like a .phony "build" target?

Reply:
Ah … make.